index.js

```diff
@@ -14,6 +14,7 @@ const Realm = require('./src/realm');
 const Decorators = require('./src/decorators');
 const Raw = require('./src/raw');
 const { MysqlDriver, PostgresDriver, SqliteDriver, AbstractDriver } = require('./src/drivers');
+const { isLeoricBone } = require('./src/utils');
```

Comment:
can we change this to `isBone()`? The `Leoric` part in the function name seems unnecessary.

Reply:
the rest looks good to me